Source/PrivateHeaders/Ably/ARTRealtimePresence+Private.h

```diff
@@ -11,6 +11,7 @@ NS_ASSUME_NONNULL_BEGIN
 - (instancetype)initWithChannel:(ARTRealtimeChannelInternal *)channel logger:(ARTInternalLog *)logger;
 - (void)_unsubscribe;
 - (BOOL)syncComplete_nosync;
+- (BOOL)syncInProgress_nosync;
```

Comment:
not a fan of this name but I can see it's already a pattern we've adopted with `syncComplete_nosync`... we can change it when we eventually migrate this to swift